cli/src/action/database/upgrade/node_id.rs

```diff
@@ -92,9 +92,19 @@ enum WarningEmited {
 #[cfg(test)]
 mod action_tests {
     use std::cell::RefCell;
+    use std::fs::File;
+    use std::io::Write;
 
     use splinter::error::InternalError;
     use splinter::node_id::store::{error::NodeIdStoreError, NodeIdStore};
+    use tempdir::TempDir;
+
+    use crate::action::database::upgrade::node_id::{
```

Comment:
Importing from the parent module in tests is one of the few areas where `use super::*` is fine.